### DISCUSSION TITLE
Sankey diagrams | Drop-Offs

### DISCUSSION THREAD

**Discussion by dberardo-com** [plouc/nivo#1800] [Q&A] [answered]
I am looking for a good library for plotting complex sankey diagrams. 

I came across Nivo's implementation which seems working quite well, but i could not figure out from the doc if and how it is possible to add drop-offs for "losses" nodes in the sankey chart.

Here an example from a proprietary library of what i am talking about:

https://www.anychart.com/products/anychart/gallery/Sankey_Diagram/User_Flow_Chart.php

Cheers

**Comment by plouc** [comment, marked as answer]
@dberardo-com, nivo sankey diagram doesn't support drop-offs, we rely on d3 sankey under the hood, and it doesn't.
